setup.py

```diff
@@ -50,6 +50,12 @@ def build_parser():
         action="store_true",
         help="""Build with Bullet simulation engine.""",
     )
+    parser.add_argument(
+        "--vhacd",
```

Comment:
Should we keep adding flags needed for the full feature suite? At some point, shouldn't bullet and VHACD become opt out params? @aclegg3 ?

Reply:
Also by not updating cirlcelci to build with VHACD, none of this code will be checked by clang-tidy?

Reply:
Yeah, that's a good point. We keep Bullet as opt-in because many standard use cases for Habitat don't require dynamics at all. We don't want to impact those users. Maybe we reconsider this once more interactive tasks hit Hab-lab. 

I think it is also ok to keep VHACD as opt-in since there are few use use cases for it at this time.

Agreed that we should add the flag to CI for testing.

Reply:
Actually, it's built twice in CI, once for clang-tidy/benchmarking and another time for testing.